.github/workflows/build.yml

```diff
@@ -0,0 +1,84 @@
+name: Build and Push to Dockerhub
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        arch: [amd64, arm64]
+    steps:
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set version to short commit SHA
+        run: |
+          shortSha=$(git rev-parse --short ${{ github.sha }})
+          echo "VERSION=$shortSha" >> $GITHUB_ENV
+
+      - name: Set up Go environment
+        uses: actions/setup-go@v5
+        with:
+          go-version: '1.21'
+          cache: false # Avoid warning: https://github.com/actions/setup-go/issues/403
```

Comment:
That issue is for v4, is this still present in v5? Caching sounds like something that's good to have (and we use this action in other repos without issues?)

Reply:
Yep, tested, it is still there, and according to the linked issue, someone tested, it is actually faster with `cache: false` somehow...